src/api/content-strings/fields/insurance/policy/index.ts

```diff
@@ -140,6 +141,13 @@ export const POLICY_FIELDS = {
         LABEL: 'Estimate the maximum amount your buyer will owe you at any single point during this time',
         HINT: {
           FOR_EXAMPLE: 'For example, your total sales might be £250,000 but the maximum the buyer will owe you at any single point is £100,000.',
+          INITIAL_CREDIT_LIMIT: {
+            INTRO: 'If your initial credit limit request is £25,000 or less you could be eligible for the',
```

Comment:
could the £25000 come from a const? incase it changes in a new version?

Reply:
Yes, would like to do this in the next PR - constants are in place there. https://github.com/UK-Export-Finance/exip/pull/3397